chart/README.md

```diff
@@ -438,7 +439,7 @@ are set e.g.
    --set ingress.annotations."alb\.ingress\.kubernetes\.io/scheme"="internet-facing"
    --set ingress.annotations."alb\.ingress\.kubernetes\.io/target-type"="ip"
    --set ingress.annotations."alb\.ingress\.kubernetes\.io/healthcheck-path"="/ping"
-   --set ingress.annotations."alb\.ingress\.kubernetes\.io/certificate-arn"="arn:aws:acm:us-east-2:119982741446:certificate/8a0a2dbf-6283-48f2-b636-4faacb0665c9"
+   --set ingress.annotations."alb\.ingress\.kubernetes\.io/certificate-arn"="arn:aws:acm:<REGION>:<AWS_ACCOUNT_ID>:certificate/<CERTIFICATE_ID>"
```

Comment:
I would make all of these `<NAME>` lower case `<name>` to follow the same pattern as earlier

Reply:
Commit: https://github.com/sonatype/nexus-iq-server-ha/commit/decb4843589915c5637baf7df6ada1cbd8bbbe1e